superset/assets/src/explore/visTypes.jsx

```diff
@@ -1247,6 +1247,7 @@ export const visTypes = {
           ['compare_lag', 'compare_suffix'],
           ['y_axis_format', null],
           ['show_trend_line', 'start_y_axis_at_zero'],
+          ['color', null],
```

Comment:
I was surprised to see that color isn't `renderTrigger: true`, looks like `color` isn't used anywhere, but `color_picker` is and behaves as expected.

Reply:
Looks like it has the right default of brandPrimary too.